demo/index.html

```diff
@@ -10,6 +10,13 @@
     <script src="libs/requirejs.min.js"></script>
     <script>
         requirejs.config({
+            config: {
```

Comment:
I think this also must be included in `demo/build.config.js`. Check if optimisation works with `npm run optimize` and open `demo/index-optimized.html` in browser.